.github/workflows/build-release-zip.yml

```diff
@@ -0,0 +1,62 @@
+name: Build release zip
+
+on:
+  workflow_dispatch:
+  workflow_call:
+
+jobs:
+  build_zip:
+    name: Build release zip
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+
+    - name: Setup node
+      uses: actions/setup-node@cdca7365b2dadb8aad0a33bc7601856ffabcc48e # v4.3.0
+      with:
+        cache: 'npm'
+        node-version-file: .nvmrc
+
+    - name: Cache node_modules
+      id: cache-node-modules
+      uses: actions/cache@d4323d4df104b026a6aa633fdb11d772146be0bf # v4.2.2
+      env:
+        cache-name: cache-node-modules
+      with:
+        path: node_modules
+        key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
+
+    - name: Get composer cache directory
+      id: composer-cache
+      run: echo "dir=$(composer config cache-files-dir)" >> $GITHUB_OUTPUT
+
+    - name: Cache dependencies
+      id: cache-composer
+      uses: actions/cache@d4323d4df104b026a6aa633fdb11d772146be0bf # v4.2.2
+      env:
+        cache-name: cache-composer
+      with:
+        path: ${{ steps.composer-cache.outputs.dir }}
+        key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/composer.lock') }}
+
+    - name: Install node dependencies
+      if: steps.cache-node-modules.outputs.cache-hit != 'true'
+      run: npm ci --no-optional
+
+    - name: Install composer dependencies
+      run: composer install --no-dev -o
+
+    - name: Build
+      run: |
+        npm run build
+        npm run makepot
+        npm run archive
```

Comment:
@s3rgiosan worth explicitly checking this section, as well as the rest of the changes in this PR, but calling this out for you

Reply:
I'll need to add those commands to the package.json file. Do you have any examples of those commands in other 10up plugins?

Reply:
https://github.com/10up/classifai/blob/develop/package.json

Reply:
@jeffpaul Added here [`3f6915f` (#94)](https://github.com/10up/wp-content-connect/pull/94/commits/3f6915f6967af51817340e4d02ff89ad00ec4d13)